cartoframes/utils/metrics.py

```diff
@@ -0,0 +1,101 @@
+import os
+import uuid
+import requests
```

Comment:
It is not a cartoframes dep (i suppose it is installed by `pyrestcli` thought `carto-python`). Should we add it to our dependencies?

Reply:
I don't think we need to add it now in our setup.py since it's a sub-dependency (for example like pandas). The only reason for that would be a specific feature from a specific version, but this is not the case now.